utils/build.ps1

```diff
@@ -148,8 +148,7 @@ param
   [string] $Stage = "",
   [ValidateSet("AMD64", "ARM64")]
   [string] $HostArchName = $(if ($env:PROCESSOR_ARCHITEW6432) { $env:PROCESSOR_ARCHITEW6432 } else { $env:PROCESSOR_ARCHITECTURE }),
-  [ValidateSet("Asserts", "NoAsserts")]
-  [string] $Variant = "Asserts",
+  [switch] $IncludeNoAsserts = $false,
```

Comment:
Is there a strong reason to make this a `-IncludeNoAsserts` rather than `-Variants asserts,noasserts`?

Reply:
Ah, that is fair. On the other hand, one thing that I did do recently is removed all the file remapping logic in the installer. This makes the installer much more straightforward - it just picks up the files from the layout as is. It does seem a shame to revert back to the old behaviour here.

Additionally, does this mean that the generated NoAsserts toolchain is not usable without building an installer and setting up a VM for testing?

Reply:
> Additionally, does this mean that the generated NoAsserts toolchain is not usable without building an installer and setting up a VM for testing?

yes. we can add a step to generate a full layout at the end. should be a simple copy. you were against that early on though

Reply:
should be updated now to copy files